src/Microsoft.Extensions.Localization/ResourceManagerStringLocalizer.cs

```diff
@@ -218,10 +229,15 @@ private IEnumerable<string> GetResourceNamesFromCultureHierarchy(CultureInfo sta
                 currentCulture = currentCulture.Parent;
             }
 
+            if (!hasAnyCultures)
+            {
+                throw new MissingManifestResourceException(Resources.FormatLocalization_MissingManifest_Parent());
```

Comment:
Only use Format when there are parameters to the resource string. Can just do: `Resources.Localization_MissingManifest_Parent`